Support more breakdown of latency metrics/stats for Llama

derekxu opened the pull request:
Summary:
Support more breakdown of latency metrics/stats for Llama
- This is needed when we debugging the Frame-LLM project across teams

Differential Revision: D64139460

cccclai reviewed (approved):
Looks good, thanks!

Commit:
Support more breakdown of latency metrics/stats for Llama (#6072)

Summary:

Support more breakdown of latency metrics/stats for Llama
- This is needed when we debugging the Frame-LLM project across teams

Reviewed By: cccclai

Differential Revision: D64139460
